Script Examples/Python/testing_example.py

```diff
@@ -1,41 +1,62 @@
 """
-Examples of unit testing.
-
-Required Items:
-    Number TestNumber1
-    Number TestNumber2
+Example of unit testing.
 """
-
 import unittest
 import time
-from core.log import logging
-from core.testing import run_test
+import core.items
+from core.jsr223 import scope, get_automation_manager
```

Comment:
These aren't used... can be removed.

Reply:
Thank you for your hint, these imports are removed. I will sign as soon as I can.